src/Pages/Login/Components/BoxForm.js

```diff
@@ -0,0 +1,176 @@
+import React from "react";
+import styled from "styled-components";
+import theme from "../../../Styles/theme";
+import Input from "./Input";
+import Button from "./Button";
+import { buttonData, inputData, spanData } from "../Data/Data";
+
+export default function BoxForm({
+  handleChange,
+  goToSignUpPage,
+  checkClicked,
+  setCheckClicked,
+  userId,
+  userPw,
+  login,
+  socialLogin,
+}) {
+  return (
+    <>
+      <InputWrapper>
+        {inputData.data.map((el, idx) => {
+          return (
+            <Input
+              key={idx}
+              name={el.name}
+              placeholder={el.text}
+              handleChange={handleChange}
+              userId={userId}
+              userPw={userPw}
+            />
+          );
+        })}
+      </InputWrapper>
+      <SpanWrapper>
+        {spanData.rememberIdPw.map((el, idx) => {
+          return (
+            <li key={idx}>
+              <i
+                className="far fa-check-square"
+                id={el.name === "rememberId" ? (checkClicked ? "boxClicked" : el.name) : el.name}
+                onClick={() => setCheckClicked(!checkClicked)}
+              />
+              <span name={el.name}>{el.text}</span>
+            </li>
+          );
+        })}
+      </SpanWrapper>
+      <LoginButtonWrapper>
+        {buttonData.loginButton.map((el, idx) => {
```

Comment:
그냥 el보다는 좀 더 직관적으로 작성해주시는 것 이 좋습니다.
아래 같은 코드들도 모두 수정해주세요!
```suggestion
        {buttonData.loginButton.map((button, idx) => {
```